src/service/performance-impl.js

```diff
@@ -783,13 +783,7 @@ export class Performance {
    * @param {string} label
    */
   mark(label) {
-    if (
-      this.win.performance &&
-      this.win.performance.mark &&
-      arguments.length == 1
```

Comment:
Do you know what the `arguments.length` check was originally for? The rest looks like a safe conversion